deploy/olm-catalog/kubevirt-hyperconverged/1.0.0/kubevirt-hyperconverged-operator.v1.0.0.clusterserviceversion.yaml

```diff
@@ -8,6 +8,7 @@ metadata:
     capabilities: Full Lifecycle
     categories: OpenShift Optional
     certified: "false"
+    olm.targetNamespaces: kubevirt-hyperconverged
```

Comment:
This is supposed to be set by olm and should not be defined in a csv. OLM will overwrite this annotation.

Reply:
> This is supposed to be set by olm and should not be defined in a csv. OLM will overwrite this annotation.

Acknowledged.
I'll remove that annotation then